modules/acquia_cms_common/modules/acquia_cms_support/acquia_cms_support.info.yml

```diff
@@ -2,6 +2,6 @@ name: "Acquia CMS Support"
 package: "Acquia CMS"
 description: "Compare the config that ships with Acquia CMS with the active config on the site."
 type: module
-core_version_requirement: ^9
+core_version_requirement: ^9 || ^10
 dependencies:
   - drupal:acquia_cms_common
```

Comment:
@chandan-singh7929 - Replace it with this acquia_cms_common:acquia_cms_common

Reply:
Done